ops/webidl/test_cases/dict.rs

```diff
@@ -13,7 +13,6 @@ pub struct Dict {
   c: Option<u32>,
   #[webidl(rename = "e")]
   d: u64,
-  f: std::collections::HashMap<String, f32>,
```

Comment:
Is IndexMap being test anywhere?

Reply:
yeah, in the `records` test in `webidl.rs`